### ISSUE TITLE
Dimmer switch cannot be turned off

### ISSUE THREAD

**Issue by attactics** [sergey-tihon/streamdeck-homebridge#13]
Switch option doesn't seem to have an off? I see on and a blank item. Looking at homebridge logs, when characteristic is set to on the light turns on, when set to the blank line, no command is sent to homebridge.

This is being used with a dimmer switch.

![image](https://user-images.githubusercontent.com/14268957/185283796-1c5d3176-e215-493d-9337-0143731d71ae.png)

**Comment by sergey-tihon** [comment]
It can be, just pick `On` from list.

Maybe UI is not very intuitive, but it says that you have assessor/device called `Light`. This device has only one binary (on/off) characteristics, the only thing that you can to turn it on or off.

If you pick `On` and press button you should turn your `Light` on, when you press it again it should go off.

**Comment by attactics** [comment]
This is what I would have assumed, but indeed when I press it again the light down not turn off and homebridge does not see an off request come in for the light.

**Comment by attactics** [comment]
This is what I would have assumed, but indeed when I press it again the
light down not turn off and homebridge does not see an off request come in
for the light.

**Comment by sergey-tihon** [comment]
Would you like to try the next version https://github.com/sergey-tihon/streamdeck-homebridge/releases/tag/1.4.0 ?
Maybe it will work for you.

**Comment by johnnygoodface** [comment]
YES! That version autoupdate the switch status on the Stream box :) Good work :)
